### DISCUSSION TITLE
Which Scopes are required for the Personal Access Token (PAT)

### DISCUSSION THREAD

**Discussion by imjared** [BetaHuhn/deploy-to-vercel-action#140] [Q&A] [answered]
Of the available scopes for a personal access token, which are required for this tool?

**Comment by BetaHuhn** [comment, marked as answer]
Hi @imjared,

The action needs to be able to access code in a repo, create/modify a PR and access deployments. To my knowledge there's no way to select only those 3, so you need the full repo scope:

![image](https://user-images.githubusercontent.com/51766171/116555158-54087f00-a8fc-11eb-9a2f-014c4b2b1804.png)

Hope this helps, let me know if you have any other questions!
